src/applications/education-letters/containers/InboxPage.jsx

```diff
@@ -0,0 +1,176 @@
+import React, { useEffect, useState } from 'react';
+import FormTitle from 'platform/forms-system/src/js/components/FormTitle';
+import { apiRequest } from 'platform/utilities/api';
+import { format } from 'date-fns';
+import { FETCH_CLAIM_STATUS } from '../actions';
+import Layout from '../components/Layout';
+
+const InboxPage = () => {
+  const [claimantId, setClaimantId] = useState(null);
+  const [isLoading, setLoading] = useState(true);
+  const [receivedDate, setReceivedDate] = useState(null);
+
+  useEffect(
+    () => {
+      const checkIfClaimantHasLetters = async () =>
+        apiRequest(FETCH_CLAIM_STATUS)
+          .then(response => {
+            setClaimantId(response?.data?.attributes?.claimantId);
+            setReceivedDate(
+              format(
+                new Date(response?.data?.attributes?.receivedDate),
+                'MMMM d, yyyy',
+              ),
+            );
+            setLoading(false);
+            return response?.data?.attributes?.claimantId;
+          })
+          .catch(err => {
+            window.location.href = '/education/education-letters/';
+            return err;
+          });
+
+      checkIfClaimantHasLetters().then(r => r);
+    },
+    [claimantId],
+  );
+
+  const HasLetters = (
+    <>
+      <FormTitle title="Your VA education letters" />
+      <p className="va-introtext">
+        Download your VA education decision letters.
+      </p>
+      <h2>Letters available for you to download</h2>
+      <div className="edu-certi-eligibility">
+        <h3 style={{ marginTop: '1rem' }}>Education decision letter</h3>
```

Comment:
Can we use design system utilities for this? https://design.va.gov/utilities/margins